src/app/pages/datasets/components/dataset-details-panel/dataset-details-panel.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="header">
-  <h3 class="title">
+  <h3 class="title {{ !(!dataset.locked && !isZvol) ? 'no-buttons' : ''}}">
```

Comment:
I noticed that you've used inversion operator `!` twice. I'd suggested to simplify, and here's 2 options:
- `dataset.locked || isZvol`
- or `(!dataset.locked && !isZvol) ? '' : 'no-buttons'`